sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -855,7 +859,9 @@ def _load_from_rest_job(cls, obj: JobBase) -> "Command":
             outputs=from_rest_data_outputs(rest_command_job.outputs),
         )
         command_job._id = obj.id
-        command_job.resources = JobResourceConfiguration._from_rest_object(rest_command_job.resources)
+        command_job.resources = JobResourceConfiguration._from_rest_object(  # type: ignore[assignment]
```

Comment:
Looks like `cast` might be an option here.

Reply:
fixed